src/Core/Tools/SendFeatures/Services/SendFileSettingHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Bit.Core.Tools.Services;
+
+namespace Bit.Core.Tools.SendFeatures;
+
+public class SendFileSettingHelper : ISendFileSettingHelper
+{
+    private readonly long _fileSizeLeeway = 1024L * 1024L; // 1MB
+    private readonly long MAX_FILE_SIZE = Constants.FileSize501mb;
+    private readonly string MAX_FILE_SIZE_READABLE = "500 MB";
+
+    public long GetMaxFileSize()
```

Comment:
⛏️ `GetMaxFileSize`, `GetMaxFileSizeReadable`, and `GetFileSizeLeeway` should be readonly properties unless there's a compelling reasonm to keep them as functions.

Reply:
I made SendFileSettingHelper into a static class since for now it will just be holding constant fields. In the future it can be a place to hold helper methods as well, but figured a static class was a better fit for the current data and can be used within the SendController without needing to add a constant variable there.